requirements-dev.txt

```diff
@@ -1,4 +1,3 @@
-setuptools-rust
```

Comment:
Just to check: is the reason for removing this just because it's specified in `pyproject.toml` as a build requirement?